backend/app/kafka/consumer_vid.py

```
@@ -0,0 +1,144 @@
+
+            # Download the video file from s3
+            local_video_path = s3_client_obj.download_from_s3(s3_BUCKET_UNTRANSCODED, video_file)
+
+            if not local_video_path:
+                send_error(video_file,f"Failed to download {video_file}. Skipping.","consumer.py")
+                print(f"Failed to download {video_file}. Skipping.")
+                continue
+                
+                
+                
+            start_t=datetime.now()
```

Ruff (0.15.15)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)